src/cmd/builtin/shutdown.c

```diff
@@ -116,6 +193,14 @@ static struct optparse_option opts[] = {
                " 0=show log messages <= LOG_INFO level (default),"
                " 1=show all log messages",
     },
+    { .name = "yes", .key = 'y', .has_arg = 0,
+      .usage = "If garbage collection threshold exceeded, "
+               "perform garbage collection",
+    },
+    { .name = "no", .key = 'n', .has_arg = 0,
+      .usage = "If garbage collection threshold exceeded, "
+               "do not perform garbage collection",
+    },
```

Comment:
How about for the usage just "answer y to all y/n questions"?

Reply:
and will tweak manpage as well